### ISSUE TITLE
Latest release makes diff command fail

### ISSUE THREAD

**Issue by aplr** [ariga/atlas#3003]
The latest release of atlas makes the diff command fail. I did not verify other commands.

### Observed behaviour

First the pipelines started to fail (no atlas cli version pin, using setup-atlas action).
After running `brew upgrade atlas`, installing the latest version, the command also started to fail.

Command:

```
atlas migrate diff "$MIGRATION_NAME" \
  --dir "file://internal/db/migrations" \
  --to "ent://internal/db/schema" \
  --dev-url "$DB_URL"
```

Result:

```
Error: .entc/entc_xxx_schema_1722199304.go:196:24: fd.Deprecated undefined (type *"entgo.io/ent/schema/field".Descriptor has no field or method Deprecated)
Error: .entc/entc_xxx_schema_1722199304.go:197:24: fd.DeprecatedReason undefined (type *"entgo.io/ent/schema/field".Descriptor has no field or method DeprecatedReason)
Error: .entc/entc_xxx_schema_1722199304.go:236:26: undefined: ent.Viewer
```

### Workaround

TL;DR: Either pin atlas to `0.25.0`, or upgrade ent to `v0.13.2-0.20240728160047-15bebe8d8920` and run ent codegen.

---

[This commit](https://github.com/ariga/atlas/commit/40e651e60db487924f0340d98928277964a45128) updated ent to `v0.13.2-0.20240728160047-15bebe8d8920` in atlas.

I did the same in my codebase, and ran the ent schema generation again.

Afterwards the `diff` command was working again.

### Versions:

- go 1.22.5
- ent 0.13.1
- atlas v0.25.1-e9f9867-canary (from brew)

**Comment by lcmmhcc** [comment]
same problem.

**Comment by a8m** [comment]
Hey! Issue was solved with ent@v0.14.1 and latest Atlas version. See: https://github.com/ariga/atlas/issues/3068#issuecomment-2308428250

We're planning to change the implementation of the ent:// loader to be (Go) module-aware. This will solve such issues in the future.
